at_lookup/lib/src/connection/outbound_sync_message_listener.dart

```diff
@@ -0,0 +1,54 @@
+import 'dart:collection';
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:at_lookup/src/connection/outbound_message_listener.dart';
+
+class SyncMessageListener extends OutboundMessageListener {
```

Comment:
you can add unit test for this class..check pendingData, verify whether sync callback is called

Reply:
Where are the tests?